FirebaseDatabase/Sources/Core/FSyncTree.m

```diff
@@ -448,6 +457,42 @@ - (NSArray *)applyTaggedServerRangeMergeAtPath:(FPath *)path
     }
 }
 
+- (NSNumber *)registerQuery:(FQuerySpec *)query {
+    FPath *path = query.path;
+
+    __block BOOL foundAncestorDefaultView = NO;
+    [self.syncPointTree
+        forEachOnPath:query.path
+           whileBlock:^BOOL(FPath *pathToSyncPoint, FSyncPoint *syncPoint) {
+             foundAncestorDefaultView =
+                 foundAncestorDefaultView || [syncPoint hasCompleteView];
+             return !foundAncestorDefaultView;
```

Comment:
I don't actually think we're using this block, since we aren't checking `foundAncestorDefaultView` later in the function

Reply:
Yes, will remove this.